src/main/java/teammates/logic/api/TaskQueuer.java

```diff
@@ -36,6 +36,12 @@ protected void addTaskMultisetParam(String queueName, String workerUrl, Map<Stri
         new TaskQueuesLogic().addTaskMultisetParam(queueName, workerUrl, paramMap);
     }
     
+    // The following methods are to be overridden by the mock object. In production, these methods are not used.
+    
+    public Map<String, Integer> getTasksAdded() {
+        return null;
```

Comment:
Would throwing `UnsupportedOperationException` be suitable here? We could explain the reason it's not used as the parameter to construct the exception, instead of leaving it as a comment above.